components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -460,10 +460,14 @@ - (CGRect)mdc_frameAlignedVertically:(CGRect)frame
 - (CGRect)mdc_frameAlignedHorizontally:(CGRect)frame
                              alignment:(MDCNavigationBarTitleAlignment)alignment {
   switch (alignment) {
-    case MDCNavigationBarTitleAlignmentCenter:
-      return CGRectMake(CGRectGetMaxX(self.bounds) / 2 - frame.size.width / 2, frame.origin.y,
-                        frame.size.width, frame.size.height);
-
+    case MDCNavigationBarTitleAlignmentCenter: {
```

Comment:
Add comment like "Align title to the center, except if X, Y, or Z." This isn't clear enough to just parse.